assets/src/js/admin/reports/pages/overview-page/index.js

```diff
@@ -3,13 +3,76 @@
 // 12 column grid for content to exist in
 
 import Grid from '../../components/grid'
+import Card from '../../components/card'
+import Chart from '../../components/chart'
 
 const OverviewPage = () => {
     return (
         <Grid>
-            <div>
-                Child Component
-            </div>
+            <Card title="Doughnut Chart" width={4}>
+                <Chart
+                    type='doughnut'
+                    aspectRatio={0.7}
+                    data={{
+                        labels: ['Jan', 'Feb', 'Mar', 'Apr', 'Jun', 'Jul'],
+                        datasets: [
+                            {
+                                label: 'Donations',
+                                data: [4, 5, 3, 7, 5, 6]
+                            }
+                        ]
+                    }}
+                />
+            </Card>
+            <Card title="Bar Chart" width={4}>
+                <Chart
+                    type='bar'
+                    aspectRatio={0.7}
+                    data={{
+                        labels: ['Jan', 'Feb', 'Mar', 'Apr', 'Jun', 'Jul'],
```

Comment:
@henryholtgeerts can you internationalize them?

Reply:
@ravinderk Thanks for the reminder about internationalization! The data object passed to the Chart component does not need to be internationalized on the frontend because it will instead be internationalized on the backend. The chart component will simply display what is returned by a call to a specified REST API endpoint, since the frontend may not know exactly what labels accompany the data (especially when querying for specific time periods). Internationalization will be handled by the methods responsible for generating the REST API response. Does that make sense?

Reply:
@henryholtgeerts Make sense